.github/workflows/CI.yml

```diff
@@ -0,0 +1,81 @@
+name: CI
+
+# Trigger the workflow on push or pull request
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: ${{ matrix.test-suites }} - ${{ matrix.branch }} - ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        test-suites: ["testinstall"]
+        branch: ["master", "stable-4.11", "stable-4.10"]
+
+    env:
+      TEST_SUITES: ${{ matrix.test-suites }}
+      GAP_PKGS_TO_CLONE: "AutoDoc"
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "Set up Python 3.7"
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: "Download CI scripts"
+        run:  git clone https://github.com/gap-system/pkg-ci-scripts.git scripts
+      - name: "Compile GAP and download packages"
+        env:
+          GAPBRANCH: ${{ matrix.branch }}
+        run:  bash scripts/build_gap.sh
+      - name: "Prepare this package"
+        run:  bash scripts/build_pkg.sh
+      - name: "Compile documentation"
+        run:  echo "QUIT;" | $HOME/gap/bin/gap.sh -l "$PWD/gaproot;" --quitonbreak makedoc.g
+      - name: "Run tests"
+        run:  bash scripts/run_tests.sh
+      - name: "Gather coverage"
+        run:  bash scripts/gather-coverage.sh
+      - name: "Upload coverage data to codecov"
+        run:  bash <(curl -s https://codecov.io/bash)
+
+  manual:
+    name: Build manuals
+    runs-on: ubuntu-latest
+
+    env:
+      GAP_PKGS_TO_CLONE: "AutoDoc"
+    steps:
+      - uses: actions/checkout@v2
+      - name: "Set up Python 3.7"
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: "Download CI scripts"
+        run:  git clone https://github.com/gap-system/pkg-ci-scripts.git scripts
+      - name: "Install latex"
+        run: |
+          packages=(
+            texlive-latex-base
+            texlive-latex-recommended
+            texlive-latex-extra
+            texlive-extra-utils
+            texlive-fonts-recommended texlive-fonts-extra
+          )
+          sudo apt-get install "${packages[@]}"
+      - name: "Compile GAP and download packages"
+        run:  bash scripts/build_gap.sh
+      - name: "Prepare this package"
+        run:  bash scripts/build_pkg.sh
+      - name: "Compile documentation"
+        run:  echo "QUIT;" | $HOME/gap/bin/gap.sh -l "$PWD/gaproot;" --quitonbreak makedoc.g
```

Comment:
Maybe also set the environment variable `SOURCE_DATE_EPOCH` to avoid timestamps in manual PDFS, which should ensure reproducible PDFs (i.e. building it twice should give identical binaries)

Reply:
That didn't work on my machine. Is there maybe a typo somewhere? I grepped for `SOURCE_DATE_EPOCH`, `DATE` and `EPOCH` in gap and in GAPDoc but didn't find anything.

Reply:
I set up a separate issues for this.